projects/kit/components/input-date-time/input-date-time.component.ts

```diff
@@ -93,6 +98,8 @@ export class TuiInputDateTimeComponent
     timeMode: TuiTimeMode = 'HH:MM';
 
     open = false;
+    dateFiller = '';
+    dateTimeFiller = '';
```

Comment:
Do we really need two separate strings here?

Reply:
deleted `dateTimeFiller `